### PR TITLE
have added the logination as google user

### PR DESCRIPTION
**Pull request by koropalov**

### CODE REVIEW

File: modules/user/user.service.js [hunk 1]
```diff
@@ -156,12 +157,13 @@ class UserService {
...
     }
-
     const user = await User.findOne({ email });
     if (!user) {
       throw new UserInputError(WRONG_CREDENTIALS, { statusCode: 400 });
     }
-
+    console.log(
+      user.credentials.find(cred => cred.source === 'horondi').tokenPass,
+    );
```
Comment (TorskyM) [inline modules/user/user.service.js:166]: Remove `console.log`

File: modules/user/user.service.js [hunk 2]
```diff
@@ -180,6 +182,63 @@ class UserService {
     };
   }
 
+  async googleUser(id_token) {
+    const client = new OAuth2Client();
+    const ticket = await client.verifyIdToken({
+      idToken: id_token,
+      audience: process.env.REACT_APP_GOOGLE_CLIENT_ID,
+    });
+    const payload = ticket.getPayload();
```
Comment (TorskyM) [inline modules/user/user.service.js:191]: Rename this variable to something like `ticketPayload`

File: modules/user/user.service.js (hunk 1 above)
Comment (koropalov) [reply modules/user/user.service.js]: Fixed

File: modules/user/user.service.js (hunk 2 above)
Comment (koropalov) [reply modules/user/user.service.js]: Fixed

**Review by INR4GE — APPROVED** [review]

**Review by ahd3r — APPROVED** [review]

**Review by TorskyM — APPROVED** [review]

**Pull request merged**